package.json

```diff
@@ -76,9 +76,11 @@
     "@typescript-eslint/eslint-plugin": "^4.15.0",
     "@typescript-eslint/parser": "^4.15.0",
     "eslint": "^7.19.0",
+    "eslint-config-prettier": "^8.3.0",
     "eslint-plugin-import": "^2.22.0",
     "eslint-plugin-prefer-arrow": "^1.2.3",
     "jest": "^27.0.0",
+    "prettier": "^2.5.1",
```

Comment:
Let's lock Prettier to an exact version like they recommend in the [installation docs](https://prettier.io/docs/en/install.html).

Reply:
Makes sense. I'll update it.

Reply:
Done.